config/generateConfigFile.cmake

```diff
@@ -50,20 +47,4 @@ if (NOT EXV_HAVE_LIBINTL_H)
     set(EXV_ENABLE_NLS 0)
 endif()
 
-
-include(CheckCSourceCompiles)
-#####################################################################################
-# strerror_r returns char*
-
-# NOTE : reverting commit #2041, which break compilation under linux and windows
-
-CHECK_C_SOURCE_COMPILES( "#include <string.h>
-int main() {
-char * c;
-c = strerror_r(0,c,0);
-return 0;
-}" EXV_HAVE_DECL_STRERROR_R )
```

Comment:
This was always evaluating to true on Linux and Mac (false on windows). It did not make sense to use it.